feature_custom_rpc/src/main/java/com/my/kizzy/feature_custom_rpc/CustomRpc.kt

```diff
@@ -167,7 +167,23 @@ fun CustomRpcScreen(
                         )
                     }
                 })
-        }) { padding ->
+        },
+        floatingActionButton = {
```

Comment:
Option to preview rpc is already present in BottomSheet

Reply:
My bad I didn't really see that :p
But the floating button is a rather convenient UX change though

![Screenshot_20241011_154033_Kizzy.jpg](https://github.com/user-attachments/assets/fc911dbf-44ed-4d84-b378-6fb78db36c3c)

I can remove it if you still think it's redundant

Reply:
Yes please remove it